pkg/services/anonymous/anonimpl/anonstore/database.go

```diff
@@ -25,7 +26,7 @@ type AnonDBStore struct {
 }
 
 type Device struct {
-	ID        int64     `json:"-" xorm:"id" db:"id"`
+	ID        int64     `json:"-" xorm:"pk autoincr 'id'" db:"id"`
```

Comment:
This is aligning with the column definition in the table: https://github.com/grafana/grafana/blob/40a1f8434dce375fb6eed382cd052cdfc52e051b/pkg/services/sqlstore/migrations/anonservice/migrations.go#L9